Sdk/Speckle.Connectors.Common/Speckle.Connectors.Common.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
why we need multi targeting for common project?

Reply:
all users of that project that are net8.0 will use the net8.0 version